src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,208 @@ set of steps, guiding toward the completion of a specified process.
 
 ## Overview
 
-Use progress indicators to keep the user on track when completing a specific
-task. By dividing the end goal into smaller, sub-tasks, it increases the
-percentage of completeness as each task is completed.
+Progress indicators help manage a user's expectations when completing a
+multistep process. They show which step the user is currently on, the total
+number of steps, and overall progress in completing a task.
 
-## Best practices
+<Row>
+<Column colLg={8}>
+
+![Example of progress indicator used in a payment UI](images/progress-indicator-usage-1.png)
+
+</Column>
+</Row>
+
+### When to use
+
+- When the user is working through a linear process that can be organized into
+  three or more steps.
+- When the user could benefit from understanding their progress on long forms
+  such as eCommerce checkouts, onboarding, or visa applications.
+- When user inputs should be validated before progressing to the next step.
+- To complement standard back/next navigation in a linear sequence.
+
+### When not to use
 
-#### Logical progression
+- When a process or form has fewer than three steps.
+- When the process may be completed in any order.
+- When the number of steps may change based on conditional logic.
 
-Display the steps in order from left to right. Indicate to the user that they
-are performing a multistep process, and show the direction of movement. Allow
-the user to return to a previous step to review their data submission.
+### Variants
 
-#### Indicate the current step
+| Variant     | Purpose                                                            |
+| ----------- | ------------------------------------------------------------------ |
+| Default     | Allows users to see where they are in a linear, multistep process. |
```

Comment:
For the words "Default" and "Interactive" in this table, we link them to the H2 heading anchor for their respective sections that are further down the page. Here is an example in the [Date picker docs](https://carbondesignsystem.com/components/date-picker/usage/#variants). And as you have probably seen, we are not always consistent with it, but I think it does help people navigate our site better to find the content they are looking for more quickly.

Reply:
Thanks! After some comment from Thy, I think we don't actually need the variant table -- just mentioning the interaction under 'Mouse' seems like enough.